### PR TITLE
Calendar

### PR DESCRIPTION
**Pull request by gabibguedes**
Adds calendar component and adds it to home with check-ins and bookings as it's events

### CODE REVIEW

File: src/helpers/parseEvents.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            start: checkin.date,
+            end: new Date(checkin.date.getTime() + checkin.duration * hour)
+        }
+    })
+
+    const bookingEvents = bookings.map( booking => {
+        return {
+            title: 'Reserva de ' + booking.bookerName,
+            start: booking.date,
+            end: new Date(booking.date.getTime() + booking.duration * hour)
```
Comment (GabrielDVpereira) [inline src/helpers/parseEvents.ts]: hey, I'm wondering if duration is in miliseconds, seconds or something. How about creating a utils like "milisecontsToHour" to parse this value here

File: src/pages/Home/index.tsx [hunk 2]
```diff
@@ -1,21 +1,37 @@
...
+import { Button, Title } from "_/components";
+import { Calendar } from "_/components/Calendar";
+import { ROUTE_PATHS } from "_/constants";
+import { useAuth, useBooking, useCheckin } from "_/contexts";
+import { parseEvents } from "_/helpers";
+import { Event } from "_/models";
 
 export function Home(){
-    const [value, setValue] = useState('')
+    const [events, setEvents] = useState([] as Array<Event>)
```
Comment (GabrielDVpereira) [inline src/pages/Home/index.tsx]: const [events, setEvents] = useState<Array<Event>>([]), you could write it like this too :)

**Review by GabrielDVpereira — COMMENTED** [review]
🏅  I left some suggestions, let me know what you think about them

File: src/helpers/parseEvents.ts (hunk 1 above)
Comment (gabibguedes) [reply src/helpers/parseEvents.ts]: done ✅

File: src/pages/Home/index.tsx (hunk 2 above)
Comment (gabibguedes) [reply src/pages/Home/index.tsx]: done ✅

**Review by GabrielDVpereira — APPROVED** [review]

**Pull request merged**